replies.go

```diff
@@ -32,19 +32,19 @@ var nightlies = "https://impactclient.net/ImpactInstaller.<EXT>?nightlies=true"
 var replies = []Reply{
 	{
 		pattern: `forge`,
-		message: "Use the [installer](https://impactclient.net/) to install Forge (1.12.2 only)\nBaritone 1.16 for Forge can be downloaded from [here](https://github.com/cabaletta/baritone/releases/download/v1.6.1/baritone-standalone-forge-1.6.1.jar)",
+		message: "Use the [installer](https://impactclient.net/) to install Forge (1.12.2 only)\nDownload standalone Baritone from [GitHub](https://github.com/cabaletta/baritone/releases).",
```

Comment:
This doesn't make it clear that baritone supports forge on versions other than 1.12 despite impact not

Reply:
Maybe something like `Standalone Baritone supports forge - download it from [GitHub]`?